src/movepick.cpp

```diff
@@ -317,30 +318,28 @@ Move MovePicker::next_move() {
 
 void MovePicker::skip_quiet_moves() { skipQuiets = true; }
 
-bool MovePicker::otherPieceTypesMobile(PieceType pt, ValueList<Move, 32>& capturesSearched) {
-    if (stage != GOOD_QUIET && stage != BAD_QUIET)
+bool MovePicker::otherPieceTypesMobile(PieceType pt) {
```

Comment:
some oversight from me when reviewing the initial patch, this function naming convention is wrong